views/main/play-settings.ejs

```diff
@@ -0,0 +1,125 @@
+<% layout('layouts/boilerplate')%>
+<div class="card">
+  <div class="card-box">
+    <div class="image-box">
+      <img
+        class="card-image"
+        id="pixelatedImage"
+        src="<%=previewImage %>"
+        alt="pixelated image"
+      />
+    </div>
+
+    <div id="sessionId" style="display: none"><%= playSessionId %></div>
```

Comment:
If you edit this value through developer tools what's gonna happen?

Reply:
Ok, well still it would be nice to have a check in backend if this value matches the one stored in mongo